definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -136,24 +110,24 @@
   {
     "LiveFrom": "12/08/2020",
     "CaseTypeID": "DIVORCE",
-    "ID": "PBAReferenceLabel",
-    "Label": "Your pay account reference: ${PBAReference}",
+    "ID": "LabelPBAReference",
+    "Label": "Your pay account reference: ${FeeAccountReference}",
     "FieldType": "Label",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "12/08/2020",
     "CaseTypeID": "DIVORCE",
-    "ID": "PBANumberLabel",
-    "Label": "Fee account number: ${PBANumber}",
+    "ID": "LabelPBANumber",
+    "Label": "Fee account number: ${SolicitorFeeAccountNumber}",
     "FieldType": "Label",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "12/08/2020",
     "CaseTypeID": "DIVORCE",
-    "ID": "HWFReferenceLabel",
-    "Label": "Fee account reference ${HWFReference}",
+    "ID": "LabelHWFReference",
+    "Label": "Fee account reference ${D8HelpWithFeesReferenceNumber}",
```

Comment:
This field also already exists: LabelSolicitorReference
<img width="576" alt="Screenshot 2020-08-17 at 11 52 05" src="https://user-images.githubusercontent.com/16866068/90388678-18849800-e080-11ea-84ef-867930a90153.png">

Reply:
Just for solicitor actually